nmigen_boards/dev/spi.py

```diff
@@ -4,16 +4,20 @@
 __all__ = ["SPIResource"]
 
 
-def SPIResource(number, *, cs, clk, mosi, miso, int=None, reset=None, attrs=None):
+def SPIResource(name, number, *, cs, clk, mosi, miso, dir="o", int=None, reset=None, attrs=None):
```

Comment:
I think dir is potentially a bad name as spi is bi-directional? Is host/device better?

Reply:
Mm yeah `role="host"` / `role="device"` would work well.

Reply:
Works for me, will change it when I get the chance.